### DISCUSSION TITLE
ratatui-org/ratatui Explore's updates crashes the application

### DISCUSSION THREAD

**Discussion by fdionisi** [community/community#75212] [Mobile]
### Select Topic Area

Bug

### Body

Whenever there is an _Activity_ update from https://github.com/ratatui-org/ratatui, or the same repository is in _Trending Repositories_, the application freezes and crashes when I scroll to the card.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.137.2 (114555847)
Device: iPhone 7 (iOS 15.7.1)
</details>

**Comment by HenSquared** [comment]
Thanks for the report @fdionisi. I am not able to reproduce currently but I have opened an issue with our team to investigate. I will keep an eye out for that repo to see what could cause it. 

Are there other repositories that cause the same issue?

**Reply by fdionisi** [reply]
Thank you @HenSquared, I'm still trying to figure out - finding this correlation with [ratatui-org/ratatui](https://github.com/ratatui-org/ratatui) has already taken me a long time. I understand that may be an old phone problem, though, so no stress. 😊
